app/components/blureable-button.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+	tagName: 'button',
+	classNameBindings: ['cssClass'],
+	/*Expected property*/
+	cssClass: '',
```

Comment:
You don't need to create a `cssClass` attribute, you can use `classNames` attribute. See http://emberjs.com/api/classes/Ember.View.html#property_classNames

This attribute is configured to be a "concatenated attribute" https://github.com/emberjs/ember.js/blob/484516eac38082ceb2ea3a7e1c0599818754849a/packages/ember-views/lib/mixins/class_names_support.js#L17

That means that when you override that attribute, values get concatenated with parent class attributes http://emberjs.com/api/classes/Ember.Object.html#property_concatenatedProperties